redis-persistence/src/main/java/com/netflix/conductor/dao/dynomite/RedisMetadataDAO.java

```diff
@@ -107,277 +104,277 @@ private void refreshTaskDefs() {
-        Preconditions.checkNotNull(eventHandler.getName(), "Missing Name");
-        EventHandler existing = getEventHandler(eventHandler.getName());
-        if(existing == null) {
-            throw new ApplicationException(Code.NOT_FOUND, "EventHandler with name " + eventHandler.getName() + " not found!");
-        }
-        index(eventHandler);
-        dynoClient.hset(nsKey(EVENT_HANDLERS), eventHandler.getName(), toJson(eventHandler));
-        recordRedisDaoRequests("updateEventHandler");
-    }
-
-    @Override
-    public void removeEventHandlerStatus(String name) {
-        EventHandler existing = getEventHandler(name);
-        if(existing == null) {
-            throw new ApplicationException(Code.NOT_FOUND, "EventHandler with name " + name + " not found!");
-        }
-        dynoClient.hdel(nsKey(EVENT_HANDLERS), name);
-        recordRedisDaoRequests("removeEventHandler");
-        removeIndex(existing);
-    }
-
-    @Override
-    public List<EventHandler> getEventHandlers() {
-        Map<String, String> all = dynoClient.hgetAll(nsKey(EVENT_HANDLERS));
-        List<EventHandler> handlers = new LinkedList<>();
-        all.entrySet().forEach(e -> {
-            String json = e.getValue();
-            EventHandler eh = readValue(json, EventHandler.class);
-            handlers.add(eh);
-        });
-        recordRedisDaoRequests("getAllEventHandlers");
-        return handlers;
-    }
-
-    private void index(EventHandler eh) {
-        String event = eh.getEvent();
-        String key = nsKey(EVENT_HANDLERS_BY_EVENT, event);
-        dynoClient.sadd(key, eh.getName());
-    }
-
-    private void removeIndex(EventHandler eh) {
-        String event = eh.getEvent();
-        String key = nsKey(EVENT_HANDLERS_BY_EVENT, event);
-        dynoClient.srem(key, eh.getName());
-    }
-
-    @Override
-    public List<EventHandler> getEventHandlersForEvent(String event, boolean activeOnly) {
-        String key = nsKey(EVENT_HANDLERS_BY_EVENT, event);
-        Set<String> names = dynoClient.smembers(key);
-        List<EventHandler> handlers = new LinkedList<>();
-        for(String name : names) {
-            try {
-                EventHandler eventHandler = getEventHandler(name);
-                recordRedisDaoEventRequests("getEventHandler", event);
-                if(eventHandler.getEvent().equals(event) && (!activeOnly || eventHandler.isActive())) {
-                    handlers.add(eventHandler);
-                }
-            } catch (ApplicationException ae) {
-                if(ae.getCode() == Code.NOT_FOUND) {}
-                throw ae;
-            }
-        }
-        return handlers;
-    }
-
-    private EventHandler getEventHandler(String name) {
-        EventHandler eventHandler = null;
-        String json = dynoClient.hget(nsKey(EVENT_HANDLERS), name);
-        if (json != null) {
-            eventHandler = readValue(json, EventHandler.class);
-        }
-        return eventHandler;
-
-    }
+	}
```

Comment:
Please avoid format change to the entire file in a feature PR in the future.

Reply:
Sure, a thought that comes to my mind is the possibility of using checkstyle plugin on gradle for the future and hook it up to the build? Hopefully changes like this will not be added anymore. Any way I will put extra attention to the code style changes.